src/components/mint-button/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './mint-button.scss';
```

Comment:
I will leave styles as-is for now to leave backwards compatibility with the current style guide. 
Will investigate css-in-js later.